test/cloudant.test.js

```diff
@@ -255,12 +268,16 @@ describe('cloudant connector', function() {
       it('returns result when sorting type provided - nested property',
         function(done) {
           CustomerSimple.find({where: {'address.state': 'CA'},
-            order: 'address.city:string DESC'},
+            order: 'address.city DESC'},
             function(err, customers) {
               if (err) return done(err);
               customers.length.should.be.equal(2);
-              customers[0].address.city.should.be.eql('San Mateo');
-              customers[1].address.city.should.be.eql('San Jose');
+              var expected1 = ['San Mateo', 'San Jose'];
+              var expected2 = ['San Jose', 'San Mateo'];
+              var actual = customers.map(function(c) {
+                return c.address.city;
+              });
+              should(actual).be.oneOf(expected1, expected2);
```

Comment:
@ssh24 sorry I am confused here, we specify the order as `address.city DESC` in query, and this test aims to make sure the result returned in the right sequence right?

But `should(actual).be.oneOf(expected1, expected2);` means you check **regardless** the order